src/e2e/specs/dashboard.spec.ts

```diff
@@ -302,6 +302,58 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard  - Payment`, ()
       `${process.env.E2E_TEST_ENV === "prod" ? "yearly" : "every ⁨2⁩ months"}`,
     );
   });
+
+  test("Verify that the Premium upsell screen is displayed correctly - overview card", async ({
+    dashboardPage,
+    // purchasePage,
+    page,
+  }) => {
+    test.info().annotations.push({
+      type: "testrail",
+      description:
+        "https://testrail.stage.mozaws.net/index.php?/cases/view/2463625",
+    });
+
+    //checking that the user can reach upsell page
+    await dashboardPage.goToDashboard();
+    const upsellButton = page.locator("section > div a");
+    await expect(upsellButton).toBeVisible();
+    await upsellButton.click();
+    await page.waitForURL(/.*\/fix\/.*\/view-data-brokers\/?/);
```

Comment:
Interesting, why does this need the optional trailing slash (`\/?`)?
Is this intermittent in the tests?

Reply:
I guess it doesn't really matter since the URL resolves to be without a '/',  but I just wanted to be sure.